### PR TITLE
Feature/KYAN-163 author bio page - add social media icons and links to author page

### PR DESCRIPTION
**Pull request by KorzunYuri**
SocialMedia sub-component was created under BlogArticleAuthorBio, serving as a container for Buttons. Resource is stored as _author/socialMedia_ node and is retrieved via AuthorInfoModel along with author info properties. BlogArticleAuthorBio can have its own socialmedia resource, but normally it will use the one under authorBioPage/jcr:content/author/socialMedia - its editing is 'delegated' to BlogArticleAuthorBio and is only allowed on author page (pages of other types are just consumers).

As we currently don't use BlogArticleAuthorBio and AuthorBioPage is a new component, we don't need any migration actions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/blogarticleauthorbio/template/.content.json [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+        "isOutlined": "false",
+        "isRounded": "false",
+        "label": "",
+        "openInNewTab": "true",
+        "openModal": "open-modal",
+        "showIconLeft": "true",
+        "showIconRight": "false",
+        "size": "is-normal",
+        "type": "a",
+        "url": "https://www.linkedin.com/company/websight-io",
```
Comment (paweljozwicki) [inline applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/blogarticleauthorbio/template/.content.json]: Let's use just https://www.linkedin.com/ ?

File: applications/blogs/backend/src/main/resources/libs/kyanite/blogs/templates/authorbiopage/initial/.content.json [hunk 2]
```diff
@@ -5,7 +5,45 @@
...
+          "isOutlined": "false",
+          "isRounded": "false",
+          "label": "",
+          "openInNewTab": "true",
+          "openModal": "open-modal",
+          "showIconLeft": "true",
+          "showIconRight": "false",
+          "size": "is-normal",
+          "type": "a",
+          "url": "https://www.linkedin.com/company/websight-io",
```
Comment (paweljozwicki) [inline applications/blogs/backend/src/main/resources/libs/kyanite/blogs/templates/authorbiopage/initial/.content.json]: Same here.

File: applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/blogarticleauthorbio/template/.content.json (hunk 1 above)
Comment (KorzunYuri) [reply applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/blogarticleauthorbio/template/.content.json]: Done

File: applications/blogs/backend/src/main/resources/libs/kyanite/blogs/templates/authorbiopage/initial/.content.json (hunk 2 above)
Comment (KorzunYuri) [reply applications/blogs/backend/src/main/resources/libs/kyanite/blogs/templates/authorbiopage/initial/.content.json]: Done, also changed it in test content

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**